icepyx/core/APIformatting.py

```diff
@@ -84,11 +84,11 @@ def _fmt_readable_granules(dset, **kwds):
                 # use single character wildcards "?" for date strings
                 # and ATLAS granule region number
                 if dset in ("ATL07", "ATL10", "ATL20", "ATL21"):
-                    granule_name = "{0}-??_{1}_{2}{3}??_*".format(dset, 14 * "?", t, c)
+                    granule_name = "{}-??_{}_{}{}??_*".format(dset, 14 * "?", t, c)
                 elif dset in ("ATL11",):
-                    granule_name = "{0}_{1}??_*".format(dset, t)
+                    granule_name = f"{dset}_{t}??_*"
                 else:
-                    granule_name = "{0}_{1}_{2}{3}??_*".format(dset, 14 * "?", t, c)
+                    granule_name = "{}_{}_{}{}??_*".format(dset, 14 * "?", t, c)
```

Comment:
Why was only 1/3 of these converted to f-string? :thinking:

Reply:
If the conversion would make the line more difficult to read then the conversion will not be made.

Reply:
Personally, I think it'd be easier to read this one as an f-string! Just my opinion though :) I'm not requesting any changes though, just curious about why it did what it did.

```python
"{}_{}_{}{}??_*".format(dset, 14 * "?", t, c)
f"{dset}_{14 * '?'}_{t}{c}??_*"
f"{dset}_??????????????_{t}{c}??_*"  # I think I like this best haha
```

Reply:
> If the conversion would make the line more difficult to read then the conversion will not be made.

Any idea what these criteria are? I'm with Matt that I think the f-strings are clearer.